RT/dt_evolve_hpsi.f90

```diff
@@ -61,7 +62,7 @@ subroutine dt_evolve_hpsi
     call update(zfac,ztpsi(:,:,tid),zu(:,ib,ik))
 
 #ifdef ARTED_CURRENT_OPTIMIZED
-    call current_omp_KB_ST(ib,ik,zu(:,ib,ik))
+    if(flag_current) call current_omp_KB_ST(ib,ik,zu(:,ib,ik))
```

Comment:
Thank for reviewing the pull request. I agree with you that the algorithm should be checked before merge.  Thus, I also asked Yabana-san to review it. I hope he will review and merge this pull request.